.env.example

```
@@ -2,5 +2,6 @@ GITHUB_APP_PRIVATE_KEY=
 GITHUB_APP_ID=
 GITHUB_WEBHOOK_SECRET=
 SMTP_PASSWORD=
+IMAGE_REGISTRY_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IMAGE_REGISTRY_PASSWORD key should go before the SMTP_PASSWORD key

(UnorderedKey)